README.md

```diff
@@ -106,6 +106,22 @@ actions: {
 {{videojs-player fluid=true}}
 ```
 
+### AspectRatio 
+
+- `aspectRatio`: This option is in the form of two integers separated by a colon like so 16:9 or 4:3
```

Comment:
This option is in the form of two integers separated by a colon: for example `16:9` or `4:3`